PR for Venue Add/Edit/Update/Delete Api calls

arevathi125 opened the pull request.

arevathi125 committed:
venue crud commtted

arevathi125 committed:
Merge branch 'main' of https://github.com/ALLHUBS-Jan-2024-Liftoff/Black-Jackets into 16-create-venue-crud

 Conflicts:
	BlackJackets API/build.gradle

arevathi125 committed:
venue crud commtted

ayjsh reviewed (commented):
Should the Venue Entity us the one that was created in the model folder?

arevathi125 committed:
Made few changes in venue

arevathi125 committed:
Committed changes in Venue

arevathi125 committed:
Committed

arevathi125 commented:
1. I have changed the entity, repo packages as models and data. 
2. Changed the data's names as well in venue. And also deleted genre and age data.
3. Changed my port as 8090 .
4. I added lombok and modelmapper dependencies in build.gradle.
5.In BlackJacketsApplication class i deleted the excluded things.

mmcadoulos reviewed (approved):
Everything looks good!
